components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -49,57 +79,57 @@ export default class GroupsList extends React.PureComponent {
             searchString: '',
         };
         Object.entries(FILTER_STATE_SEARCH_KEY_MAPPING).forEach(([key]) => {
-            this.state[key] = false;
+            (this.state as any)[key] = false;
         });
     }
 
-    closeFilters = () => {
+    public closeFilters(): void {
```

Comment:
Non-blocking, but the `void` type isn't required here.

Reply:
removed void